docs/control_center_docs/speed_module_getting_started.rst

```diff
@@ -0,0 +1,166 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _speed_module_start_guide:
+
+####################################################################
+Getting Started with Vertiq's Speed Firmware with IQ Control Center
+####################################################################
+
+Before completing the following getting started guide, please ensure that you have read and completed our :ref:`IQ Control Center 
+guide <control_center_start_guide>`. It walks you through Control Center installation, module configuration, and basic testing options available 
+to all Vertiq modules and firmware types. The following document is meant to provide additional configurations and testing options available only on Vertiq's speed firmware.
+
+.. note::
+    The following images are captured using IQ Control Center version 1.5.2 as connected to a Vertiq 40-06 Gen 2
+
+*********************************************************
+Important Parameters for Flight Controller Integration
+*********************************************************
+
+The first set of configurations covered here are those that **must** be configured properly in order to control your module with a flight controller. 
+These parameters affect how your module communicates with, and is controlled by a flight controller. 
+
+* **Mode**: this parameter is available through the General tab, and defines whether your module accepts throttle commands as a voltage, a velocity, or a PWM. 
+  To learn more about the Mode parameter see :ref:`throttle_mode`
+* **Direction**: this parameter is available through the General tab, and defines the direction that your module takes to be positive, either clockwise or 
+  counter-clockwise. You can also configure either 2D or 3D configuration to allow for mapping negative throttle commands.
+  For more on the *Direction* parameter and throttle mapping, see :ref:`throttle_direction`
+* **FC 2D/3D Mode**: This parameter is available through the General tab, and defines how ESC commands will be mapped to a throttle. To learn more see :ref:`throttle_mapping`
+* **Communication**: This parameter is available through the General tab, and defines the communication protocol expected by your module as sent by the 
```

Comment:
In this first sentence, can you just add "defines the communication protocol expected by your module as sent by the flight controller **when using hobby protocols**". As it is, this first sentence is misleading, and then only gets clarified at the end